packages/dev/core/src/Meshes/GaussianSplatting/gaussianSplattingMesh.ts

```diff
@@ -75,6 +79,39 @@ export class GaussianSplattingMesh extends Mesh {
         return this._colorsTexture;
     }
 
+    /**
+     * set rendering material
+     */
+    public override set material(value: Material) {
+        this._material = value;
+        this._material.backFaceCulling = true;
+        this._material.cullBackFaces = false;
+        this._clearObservable();
+        if (value.getClassName() === "NodeMaterial") {
+            value.onBindObservable.add(this._materialBindCallback, undefined, undefined, this);

```

Comment:
I think it would be cleaner to do the bindings this way:

https://github.com/Popov72/Babylon.js/commit/916b08e80725ff49afd03e5a01a0bae1ce459e40

Reply:
It looks better, indeed!